Makefile

```diff
@@ -73,12 +74,12 @@ down: ## remove stack (warning: it removes the database container)
 	@$(COMPOSE) down || echo WARNING: unable to remove the stack. Try to stop linked containers or networks first.
 .PHONY: down
 
-format: ## format Jsonnet sources
-	bin/jsonnetfmt -i $(sources)
+format: ## format Jsonnet sources and libraries
+	bin/jsonnetfmt -i $(sources) $(libraries)
```

Comment:
We also need to update the CI to that libraries are also checked.

Reply:
Well spotted! Thanks) Updated the circleci to use `make format`)